### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by PrathmeshMutke**
Commit Message:
Additional Description:
Risk Level:
Testing:
Docs Changes:
Release Notes:
Platform Specific Features:
[Optional Runtime guard:]
[Optional Fixes #Issue]
[Optional Deprecated:]
[Optional [API Considerations](https://github.com/envoyproxy/envoy/blob/main/api/review_checklist.md):]

### CODE REVIEW

**Comment by jmarantz** [comment]
Hi -- before we start reviewing, can you fix DCO?

See https://github.com/envoyproxy/envoy/blob/main/CONTRIBUTING.md#dco-sign-your-work

Thanks!

**Comment by PrathmeshMutke** [comment]
> Hi -- before we start reviewing, can you fix DCO?
> 
> See https://github.com/envoyproxy/envoy/blob/main/CONTRIBUTING.md#dco-sign-your-work
> 
> Thanks!
Hey I am newly joined here will you explain how to do it

**Comment by PrathmeshMutke** [comment]
Signed-off-by: PrathmeshMutke<pmutke03@gmail.com>

File: docs/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Building documentation locally
 
-There are two methods to build the documentation, described below.
+There are two methods to build the documentation, described below :
```
Comment (jmarantz) [inline docs/README.md:3]: drop the space before the ":" ?

Comment (PrathmeshMutke) [reply docs/README.md:3]: yes

**Comment by jmarantz** [comment]
Unfortunately DCO is pickier -- can you fully read the link I provided? I think you'll need to either do a force-push to erase all non-DCO'd commits, or you'll have to open a new PR. This is a hassle during your first PR but once you have the git hooks sorted out it'll just work always.

/wait

**Pull request closed without merging**